random.c

```diff
@@ -1296,6 +1296,21 @@ random_s_bytes(VALUE obj, VALUE len)
     return rand_bytes(&random_mt_if, rnd, NUM2LONG(rb_to_int(len)));
 }
 
+/*
+ * call-seq: Random.seed -> integer
+ *
+ * Returns the seed value used to initialize the generator. This may be used to
+ * initialize another generator with the same state at a later time, causing it
+ * to produce the same sequence of numbers.
+ *
+ *   Random.seed      #=> 1234
+ *   prng1 = Random.new(Random.seed)
+ *   prng1.seed       #=> 1234
+ *   prng1.rand(100)  #=> 47
+ *
+ *   prng2 = Random.new(Random.seed)
+ *   prng2.rand(100)  #=> 47
```

Comment:
“The same results for the same seed” is a spec of `Random#seed`, not `Random.seed`.
I think what should be described here are the value is not affected by `prng1.rand` and the next value from `Random` equals the `prng1`'s result.
```suggestion
 *   Random.seed      #=> 1234
 *   Random.rand(100) #=> 47
```